### ISSUE TITLE
It is not allowed to repeat the same properties when rewriting a third-party library :hover 

### ISSUE THREAD

**Issue by yoyo837** [stylelint/stylelint#3468]
It report: `declaration-block-no-duplicate-properties`.  `color`

What should I do?

```less
.arrow {
    font-size: 32px;
    padding: 0;
    height: @base;
    width: @base;
    color: #1890ff;
    top: 0;
    margin: 0;
    color: @primary-color;
    &:hover {
      color: @primary-color;
    }
  }
```

**Comment by yoyo837** [comment]
Sorry, my bad.
